Pipelines/Templates/tests.yaml

```diff
@@ -78,50 +45,49 @@ steps:
-        Write-Output '====================================================='
-
-        Get-Content $logFile
-
-        Write-Output '====================================================='
-        Write-Output '              End PlayMode Tests Logs                '
-        Write-Output '====================================================='
-      }
-      else {
-        Write-Output 'PlayMode Tests log missing!'
       }
-
-      if (Test-Path $resultFile) {
+      finally {
         Write-Output '====================================================='
-        Write-Output '            Begin PlayMode Tests Results             '
-        Write-Output '====================================================='
-
-        Get-Content $resultFile
-
-        Write-Output '====================================================='
-        Write-Output '             End PlayMode Tests Results              '
+        Write-Output '               PlayMode Tests Finished               '
         Write-Output '====================================================='
+        if (Test-Path $logFile) {
+          Write-Output '====================================================='
+          Write-Output '             Begin PlayMode Tests Logs               '
+          Write-Output '====================================================='
+          Get-Content $logFile
+          Write-Output '====================================================='
+          Write-Output '              End PlayMode Tests Logs                '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests log missing!'
+        }
+        if (Test-Path $resultFile) {
+          Write-Output '====================================================='
+          Write-Output '            Begin PlayMode Tests Results             '
+          Write-Output '====================================================='
+          Get-Content $resultFile
+          Write-Output '====================================================='
+          Write-Output '             End PlayMode Tests Results              '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests results missing!'
+        }
+      Write-Output $($proc.ExitCode)
       }
-      else {
-        Write-Output 'PlayMode Tests results missing!'
-      }
-
-      Write-Output "Exit code: $($proc.ExitCode)"
-    }
-  displayName: 'Run PlayMode tests'
 
-- task: PublishTestResults@2
-  displayName: 'Publish Test Results'
-  inputs:
-    testResultsFormat: NUnit
-    testResultsFiles: 'test*.xml'
-    searchFolder: $(Agent.TempDirectory)
-    failTaskOnFailedTests: true
+    displayName: 'Run PlayMode tests'
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: 'D:\a\_work\1\a'
+      Contents: '**'
+      TargetFolder: $(Build.ArtifactStagingDirectory)\Temp
```

Comment:
Will the above pwsh script still work if we have Unity put the xml file in this Temp folder directly instead of adding a copy?

Also, does this command as-written copy the entire project into this Temp directory? Seems like a lot for just one file! 😅